box2d/src/test/kotlin/ktx/box2d/worldsTest.kt

```diff
@@ -171,4 +171,44 @@ class WorldsTest : Box2DTest() {
     assertSame(body.fixtureList[0], callbackParameter)
     world.dispose()
   }
+
+  @Test
+  fun `should ray-cast between two vectors`() {
+    val world = createWorld()
+
+    val expectedEdge = world.body {}.edge(from = Vector2.Zero, to = Vector2(0f, 2f)) {}
```

Comment:
I chose to call `edge()` on the returned body because the test needs a reference to the created Edge fixture.

I couldn't see an alternative, unless I make `var expectedEdge` or a callback, but both those options don't seem ideal.